app/models/api/hses.rb

```diff
@@ -1,25 +1,49 @@
+require "api_request"
 require "fake_api_response_wrapper"
 require "fake_issues"
 
-class Api::Hses
-  class Issue
-    include FakeApiResponseWrapper
-    attr_accessor :id
+module Api::Hses
```

Comment:
[thought] i think if we keep this a class we can have it inherit from ApiRequest so every subclass doesn't need to. thoughts?

Reply:
I agree that would be nice, but I was having trouble figuring out how to keep something namespaced while also having inheritance, since those aren't the same thing.

Like, `Api::Hses` is different than `Hses < Api` but I don't really want to say `Api::Hses < Api` ... although I might just be missing something about how to have my cake and inherit some, too.

Reply:
But it would be `Api::Hses < ApiRequest` which seems okay to me

Reply:
As far as I'm aware there aren't really "nested" classes in Ruby, so I don't think you can inherit methods through namespacing / nesting. I tried this out just now to check, and didn't get the error I was expecting so maybe there a way to access the methods after all, but I will need to look into it more:

```ruby
class ApiRequest
  def test
    puts "from ApiRequest"
  end
end

class Api
  class Hses < ApiRequest
    class Issue
    end
  end
end

ApiRequest.new.test
Api::Hses::Issue.new.test
```

> from ApiRequest
> testing.rb:18:in `<main>': private method `test' called for #<Api::Hses::Issue:0x00007fd55f8d39e8> (NoMethodError)

Reply:
eh ok, no biggie